yellowbrick/utils/nan_warnings.py

```diff
@@ -0,0 +1,31 @@
+"""
+Small helpers that help find missing data.
+"""
+
+import numpy as np
+import warnings
+
+
+def warn_if_nans_exist(data):
+    """Warn if nans exist in a numpy array."""
+    null_count = count_nan_rows(data)
+    total = len(data)
+    percent = null_count / total
+
+    # if np.isnan(data).any():
+    if null_count > 0:
+        warnings.warn(
+            'Warning! Found {} rows of {} ({}%) with nan/null/None '
+            'values which are cannot be plotted.'.format(null_count, total,
+                                                         percent))
+
+
+def count_nan_rows(data):
+    """Count the number of rows that contain any nan values."""
+    if data.shape[0] >= 2:
```

Comment:
Out of curiosity, why do we condition our count to require a minimum of two rows?

Reply:
Nice catch. This was a late night error meant to check for 2D arrays. 😄